tedana/decomposition/eigendecomp.py

```diff
@@ -167,20 +168,24 @@ def tedpca(catd, OCcatd, combmode, mask, t2s, t2sG, stabilize,
         o[mask] = eimum
         eimum = np.squeeze(o).astype(bool)
 
-        vTmix = v.T
+        vTmix = comp_ts.T
         vTmixN = ((vTmix.T - vTmix.T.mean(0)) / vTmix.T.std(0)).T
         LGR.info('Making initial component selection guess from PCA results')
-        _, ctb, betasv, v_T = model.fitmodels_direct(catd, v.T, eimum, t2s, t2sG,
-                                                     tes, combmode, ref_img,
-                                                     mmixN=vTmixN, full_sel=False)
-        ctb = ctb[ctb[:, 0].argsort(), :]
-        ctb = np.vstack([ctb.T[:3], sp]).T
+        _, ct_df, betasv, v_T = model.fitmodels_direct(catd, comp_ts.T, eimum,
+                                                       t2s, t2sG,
+                                                       tes, combmode, ref_img,
+                                                       mmixN=vTmixN,
+                                                       full_sel=False)
+        ct_df['variance explained 2 (normalized)'] = varex_norm
```

Comment:
I don't love `variance explained 2 (normalized)`. Could we just do `normalized variance explained` ? Is there a reason to keep the `2` ?

Reply:
There is a normalized variance explained in `fitmodels_direct`, although I don't think it's ever used:

https://github.com/tsalo/tedana/blob/869b0842cb95b21f67cd79bd08dd995db6e48c39/tedana/model/fit.py#L207-L211

And then there's the second one we see above in `tedpca`. `variance explained (normalized)` is on a percentage scale (0-100), while `variance explained 2 (normalized)` is on a 0-1 scale. Even accounting for that, though, they aren't the same.  I can definitely rename one or both of them, and it might be worth it to drop the one in `fitmodels_direct`.